servers/src/test/java/tachyon/worker/block/TieredBlockStoreTestUtils.java

```diff
@@ -77,40 +90,118 @@ public static TachyonConf newTachyonConf(int[] tierLevel, StorageLevelAlias[] ti
+   *
+   * @param baseDir the directory path as prefix for all the paths of directories in the tiered
+   *        storage. When specified, the directory needs to exist before calling this method.
+   * @param tierLevel level of this tier.
+   * @param tierAlias alias of this tier.
+   * @param tierPath path of this tier. When `baseDir` is specified, the actual test tierPath
+   *        turns into `baseDir/tierPath`.
+   * @param tierCapacity capacity of this tier.
+   * @param workerDataFolder when specified it sets up the tachyon.worker.data.folder property.
+   * @return the created TachyonConf.
+   * @throws Exception when error happens during creating temporary folder.
+   */
+  public static void setTachyonConfWithSingleTier(String baseDir, int tierLevel,
+      StorageLevelAlias tierAlias, String[] tierPath, long[] tierCapacity, String
+      workerDataFolder) throws Exception {
+    if (baseDir != null) {
+      tierPath = concatAndCreateDir1D(baseDir, tierPath);
+    }
+    if (workerDataFolder != null) {
+      sTachyonConf.set(Constants.WORKER_DATA_FOLDER, workerDataFolder);
+    }
+    sTachyonConf.set(Constants.WORKER_MAX_TIERED_STORAGE_LEVEL, String.valueOf(1));
+    setTachyonConfOneTier(tierLevel, tierAlias, tierPath, tierCapacity);
+  }
+
+  /**
+   * Setup a specific tier's {@link TachyonConf} for a {@link TieredBlockStore}.
+   *
+   * @param level level of the tier.
+   * @param tierAlias alias of the tier.
+   * @param tierPath absolute path of the tier.
+   * @param tierCapacity capacity of the tier
+   */
+  private static void setTachyonConfOneTier(int level, StorageLevelAlias tierAlias,
+      String[] tierPath, long[] tierCapacity) {
+    Preconditions.checkNotNull(tierPath);
+    Preconditions.checkNotNull(tierCapacity);
+    Preconditions.checkArgument(tierPath.length == tierCapacity.length,
+        String.format("tierPath and tierCapacity should have the same length"));
+
+    sTachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_ALIAS_FORMAT, level),
+        tierAlias.toString());
+
+    String tierPathString = StringUtils.join(tierPath, ",");
+    sTachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_PATH_FORMAT, level),
+        tierPathString);
+
+    String tierCapacityString = StringUtils.join(ArrayUtils.toObject(tierCapacity), ",");
+    sTachyonConf.set(String.format(Constants.WORKER_TIERED_STORAGE_LEVEL_DIRS_QUOTA_FORMAT, level),
+        tierCapacityString);
+  }
+
+  /**
+   * Concat baseDir with all the paths listed in the array and then create the new generated path.
+   *
+   * @param baseDir the directory path as prefix for all the paths in the array 'dirs'.
+   * @param dirs 2-D array of directory paths.
+   * @return new joined and created paths array.
+   * @throws Exception when error happens during creating temporary folder
+   */
+  private static String[][] concatAndCreateDir2D(String baseDir, final String[][] dirs)
```

Comment:
There is no need for the method name to try to capture the details of what the function is doing. That's what the method description is for. I would just use ``createDirHierarchy``.